engine/lite/embeddedDeploymentManager/src/main/scala/pl/touk/nussknacker/engine/embedded/requestresponse/RequestResponseDeploymentStrategy.scala

```diff
@@ -34,22 +38,30 @@ object RequestResponseDeploymentStrategy {
     new RequestResponseDeploymentStrategy(config.as[HttpConfig]("http"), config.as[RequestResponseConfig]("request-response"))
   }
 
+  def slugForScenario(metaData: MetaData): Validated[NonEmptyList[FatalUnknownError], String] = metaData.typeSpecificData match {
+    case RequestResponseMetaData(slug) => Valid(slug.getOrElse(defaultSlug(ProcessName(metaData.id))))
+    case _ => Invalid(NonEmptyList.of(FatalUnknownError(s"Wrong scenario metadata: ${metaData.typeSpecificData}")))
+  }
+
+  // TODO: make it compatible with k8s version
+  def defaultSlug(scenarioName: ProcessName): String = scenarioName.value
```

Comment:
we don't want some escaping here?

Reply:
I've added "sanitization" instead - it will be better presented than escaped url